integration/README.md

```diff
@@ -0,0 +1,305 @@
+```sh
+$ curl -sO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-docker/master/examples/docker-file.json
+$ snaptel task create -t ./docker-file.json
+``` 
+Command:
+```sh
+$ snaptel task list
+``` 
+will provide information whether the metrics collection was successfull or not. If you want to know how to load other plugins read the section [Customizing Snap Dockerfile and Kubernetes manifest for specific plugins](#5-cutomizing-snap-dockerfile-and-kubernetes-manifests-for-specific-plugins).
+
+### 3. Running Snap on Kubernetes cluster
+To run Snap in Kubernetes pods create daemonset from manifest file `snap/integration/kubernetes/snap.yaml`.
+```sh
+$ kubectl create -f snap/integration/kubernetes/snap.yaml
+```
+Verify that pods have been created. 
+```sh
+$ kubectl get pods --namespace=kube-system
+```
+Log into the one of pods using the pod name returned by `kubectl get pods` command.
+```sh
+$ kubectl exec -ti <snap-pod-name> bash --namespace=kube-system
+```
+Now that you are inside the pod with running Snap daemon, you may run command to list the plugins
+```sh
+$ snaptel plugin list
+``` 
+The output should be `No plugins found. Have you loaded a plugin?` as there are no plugins loaded yet. Let's download some...
+```sh
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-collector-docker/releases/download/5/snap-plugin-collector-docker_linux_x86_64" -o snap-plugin-collector-docker
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64" -o snap-plugin-publisher-file
+```
+...and load them.
+```sh
+$ snaptel plugin load snap-plugin-collector-docker
+$ snaptel plugin load snap-plugin-publisher-file
+```
+Now, running the command
+```sh
+$ snaptel plugin list
+``` 
+will print information about two loaded plugins. To start the collection of metrics you can create task.
+```sh
+$ curl -sO https://raw.githubusercontent.com/intelsdi-x/snap/integration/integration/docker-file.json
+$ snaptel task create -t ./docker-file.json
+``` 
+Command:
+```sh
+$ snaptel task list
+``` 
+will provide information whether the metrics collection was successfull or not. Use task watch to verify that tasks have been created correctly.
+```sh
+$ snaptel task watch <task-id>
+```
+If you want to know how to load other plugins read the section [Customizing Snap Dockerfile and Kubernetes manifest for specific plugins](#5-cutomizing-snap-dockerfile-and-kubernetes-manifests-for-specific-plugins).
+
+### 4. Running Snap on Kubernetes with Heapster publisher
+There is also a possibility to publish metrics gathered by Snap collector to Heapster (https://github.com/kubernetes/heapster). This solution requires Snap Heapster publisher (https://github.com/intelsdi-x/snap-plugin-publisher-heapster) and running customized Heapster Docker image with Snap source added so that Heapster can scratch metrics directly from running Snap instances.
```

Comment:
"...to Heapster" -> "... to customized Heapster (supporting Snap as a data source)"